.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: ci
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    types: [ opened, reopened ]
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+          cache: 'pip'
+      - name: Install dependencies
+        run: |
+          # python -m pip install --upgrade pip
+          python -m pip install --upgrade pip setuptools wheel
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
let's install https://pypi.org/project/flake8-import-order/ to check order of imports